wolf/media/gst/audio/w_audio_info.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "wolf.hpp"
+
+#include "media/gst/internal/w_common.hpp"
+#include "media/gst/internal/w_wrapper.hpp"
+#include "media/gst/core/w_caps.hpp"
+#include "media/gst/audio/w_audio_format.hpp"
+
+#include <gst/audio/audio-info.h>
+
+#include <stdexcept>
+#include <utility>
+
+namespace wolf::gst {
```

Comment:
please change it to wolf::media::gst

Reply:
done at https://github.com/WolfEngine/WolfEngine/pull/452/commits/e15b76708175917332bb40fbd92cc7b3f6965d17.